tests/draft6/optional/format.json

```diff
@@ -47,6 +52,55 @@
         ]
     },
     {
+        "description": "validation of URI References",
+        "schema": {"format": "uri-reference"},
+        "tests": [
+            {
+                "description": "a valid URI",
+                "data": "http://foo.bar/?baz=qux#quux",
+                "valid": true
+            },
+            {
+                "description": "a valid protocol-relative URI Reference",
+                "data": "//foo.bar/?baz=qux#quux",
+                "valid": true
+            },
+            {
+                "description": "a valid relative URI Reference",
+                "data": "/abc",
+                "valid": true
+            },
+            {
+                "description": "an invalid URI Reference",
+                "data": "\\\\WINDOWS\\fileshare",
+                "valid": false
+            },
+            {
+                "description": "a valid URI Reference",
+                "data": "abc",
+                "valid": true
+            }
+        ]
+    },
+    {
+        "description": "format: uri-template",
```

Comment:
Yeah it's not obvious but it's there.  It just means that the template can look like a relative uRI reference in addition to a full URI.

At the top of page 3 in the introduction there's this:

>    template processor:  A program or library that, given a URI Template
      and a set of variables with values, transforms the template string
      into a **URI reference** by parsing the template for expressions and
      substituting each one with its corresponding expansion.

(emphasis added).

In the expansion section:

>   If an error occurs, the result returned might not be a valid **URI
   reference**; it will be an incompletely expanded template string that
   is only intended for diagnostic use.

It's mentioned in several other places but I think that gets the point across :-)